src/App.tsx

```diff
@@ -1,24 +1,67 @@
 import * as React from 'react';
-import logo from './logo.svg';
+
 import styles from './App.module.scss';
+import * as api from './api';
+import Login from './components/Login';
+
+interface Props {}
```

Comment:
I believe we decided to use `type` for everything, except when `interface` is specifically needed. I'm going to update my open PR to make that change and it should probably be reflected in this PR as well.

Reply:
agreed